package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rg-bot",
-  "version": "1.9.0",
+  "version": "1.10.0",
```

Comment:
You should be able to leave this at whatever I have it in the newest package json, since I haven't published yet.

Reply:
ok this time, but i don't agree with having to trust/know that you won't publish something that's already merged before mine gets published/merged.. that seems like we're playing with fire

Reply:
actually, I added apis ,shouldn't this be a minor update after all ?

Reply:
I think we need to put together some docs on how we version... for instance, for rg-service, we don't update the version on every commit, only every release...

Happy with this being a minor update and publishing for now.